alacritty_terminal/src/grid/storage.rs

```diff
@@ -66,7 +66,7 @@ impl<T> Storage<T> {
     #[inline]
     pub fn with_capacity(visible_lines: usize, columns: usize) -> Storage<T>
     where
-        T: Clone + Default,
```

Comment:
This could be reverted? I think as well as other changes to `alacritty_terminal`?

Reply:
Why? I think it's better to have more accurate constraints. There's no reason why we'd require clone if we don't need it.

Reply:
If you want to lower it in general, I'm fine with it. Maybe separate commit then, since these changes have nothing to do with the issue we're solving.

Reply:
The changes were made in a separate commit, yes.